fcrepo-exts/migration-utils
# Fix Deleted and Inactive migration

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3211

# What does this Pull Request do?

* Objects and datastreams that are in a Deleted state are migrated and then removed from the head version
* By default Inactive objects and datastreams are migrated the same as Active objects and datastreams
* A flag, `-I`, was added to migrate Inactive objects and datastreams the same as Deleted objects and datastreams
* If an object is in a Deleted state, then this state also applies to all of its datastreams

# How should this be tested?

Migrate F3 content that contains Deleted or Inactive resources. See that they are not present in the head version, but are in earlier versions. Start F6 on the migrated content.

# Interested parties
@fcrepo/committers
